app/code/Magento/ImportExport/Model/History.php

```diff
@@ -59,7 +59,7 @@ public function __construct(
         \Magento\ImportExport\Model\ResourceModel\History $resource,
         \Magento\ImportExport\Model\ResourceModel\History\Collection $resourceCollection,
         \Magento\ImportExport\Helper\Report $reportHelper,
-        \Magento\Backend\Model\Auth\Session $authSession,
+        \Magento\Backend\Model\Auth\Session\Proxy $authSession,
```

Comment:
Proxy should never be requested directly from the constructor. Instead, you should configure DI via di.xml to substitute Proxy for the authSession parameter.